### DISCUSSION TITLE
Fill_between functionality?

### DISCUSSION THREAD

**Discussion by Svalorzen** [alandefreitas/matplotplusplus#78] [Q&A] [answered]
I'm looking to plot some lines with uncertainty estimate (standard deviation), and I'm used to the `fill_between` function in matplotlib. I've looked in the repo and grepped the code but could not find anything similar to this, and in the examples the only one that fills is the polygon one.

Is this functionality planned, or is there a simple way to emulate it? The idea would be to simply pass two lines and have the space between them be shaded.

**Comment by alandefreitas** [comment, marked as answer]
Maybe the second example here can help you: https://alandefreitas.github.io/matplotplusplus/examples/line-plots/error-bars.html

**Reply by Svalorzen** [reply]
Thanks, it's pretty much what I was looking for, I didn't see it before. Any chance you'll allow non-symmetric error bars at some point?

**Reply by alandefreitas** [reply]
There are some non-symmetric error bars in these examples if i remember correctly. For instance, https://github.com/alandefreitas/matplotplusplus/blob/master/examples/line_plot/errorbar/errorbar_7.cpp. Then you just need to use `filled_curve` like in https://github.com/alandefreitas/matplotplusplus/blob/master/examples/line_plot/errorbar/errorbar_2.cpp if you want filled curves to replace the bars.
